python_modules/dagster/dagster/_generate/generate.py

```diff
@@ -3,36 +3,86 @@
 
 import jinja2
 
+from dagster._utils import file_relative_path
 from dagster.version import __version__ as dagster_version
 
-NEW_PROJECT_PLACEHOLDER = "new_project"
-NEW_PROJECT_PATH = os.path.join(os.path.dirname(__file__), NEW_PROJECT_PLACEHOLDER)
+
+def generate_repository(path: str):
```

Comment:
Not a blocker or anything, but it would be cool for this to add an entry to the workspace.yaml

Reply:
yup and add an entry to the requirements.txt in the project root as well.
in a later diff, i plan to a cli prompt to ask for paths to workspace.yaml and requirements.txt - it could be tricky to assume the file path or name.